toucan_connectors/awsathena/awsathena_connector.py

```diff
@@ -0,0 +1,45 @@
+import awswrangler as wr
+import boto3
+import pandas as pd
+from pydantic import Field, constr
+
+from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource
+
+
+class AwsathenaDataSource(ToucanDataSource):
+    name: str = Field(..., description='Your AWS Athena connector name')
+    database: constr(min_length=1) = Field(
+        ..., description='The name of the database you want to query.'
+    )
+    query: constr(min_length=1) = Field(
+        ...,
+        description='The SQL query to execute.',
+        widget='sql',
+    )
+
+
+class AwsathenaConnector(ToucanConnector):
+    data_source_model: AwsathenaDataSource
+
+    name: str = Field(..., description='Your AWS Athena connector name')
+
+    s3_output_bucket: str = Field(
+        ..., description='Your S3 Output bucket (where query results are stored.)'
+    )
+    aws_access_key_id: str = Field(..., description='Your AWS access key ID')
+    aws_secret_access_key: str = Field(..., description='Your AWS secret key')
```

Comment:
```suggestion
    aws_secret_access_key: SecretStr = Field(None, description='Your AWS secret key')
```
Le type à utiliser ici est SecretStr, qui permet de stocker cette information encrypter dans la collection `connection_secrets` dédiée de mongo. 
Le champs doit être à None à cause de notre logique de sauvegarde des queries qui fait qu'on valide la config d'un connecteur à la sauvegarde d'une query (mais on a plus la SecretStr à ce moment là).

Reply:
Ok merci! Même en passant le défaut à `None` le champ sera marqué comme requis dans le front ?

Et on va faire le changement pour l'access key ID tant qu'à faire

Reply:
Non malheureusement il ne sera pas indiqué en requis :S